From 1ae91887dbcafb2327dc320f606ffd18de9e27aa Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 4 Jul 2017 11:22:15 +0800
Subject: [PATCH 2038/5242] MLK-15328-1 input: keyboard: add i.mx8mq snvs
 onoff button support

commit  8dedcbfcbb854380e3fae0dba90b5ce28f6b5598 from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX8MQ use SNVS ONOFF button, add support for it.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/input/keyboard/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/keyboard/Kconfig b/drivers/input/keyboard/Kconfig
index d8bea47..b4b3320 100644
--- a/drivers/input/keyboard/Kconfig
+++ b/drivers/input/keyboard/Kconfig
@@ -429,7 +429,7 @@ config KEYBOARD_RPMSG
 
 config KEYBOARD_SNVS_PWRKEY
 	tristate "IMX SNVS Power Key Driver"
-	depends on (SOC_IMX6SX || SOC_IMX6UL || SOC_IMX7)
+	depends on (SOC_IMX6SX || SOC_IMX6UL || SOC_IMX7 || ARCH_FSL_IMX8MQ)
 	depends on OF
 	help
 	  This is the snvs powerkey driver for the Freescale i.MX application
-- 
1.7.9.5

